pkg/registry/apps/statefulset/storage/storage.go

```diff
@@ -92,3 +115,88 @@ var _ rest.ShortNamesProvider = &REST{}
 func (r *REST) ShortNames() []string {
 	return []string{"sts"}
 }
+
+type ScaleREST struct {
+	registry statefulset.Registry
+}
+
+// ScaleREST implements Patcher
+var _ = rest.Patcher(&ScaleREST{})
+
+// New creates a new Scale object
+func (r *ScaleREST) New() runtime.Object {
+	return &extensions.Scale{}
```

Comment:
so, it's kind of even worse that that.  In order to actually work, there has to be an object in the extensions category, so this storage basically doesn't matter.  I'd do what the RC storage does (return `autoscaling.Scale` from "primary" group, for use in the future, and then have the extensions storage return `extensions.Scale`)